raven/src/main/java/com/getsentry/raven/RavenContext.java

```diff
@@ -0,0 +1,117 @@
+package com.getsentry.raven;
+
+import com.getsentry.raven.event.Breadcrumb;
+import com.getsentry.raven.util.CircularFifoQueue;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.IdentityHashMap;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * RavenContext is used to hold {@link ThreadLocal} context data (such as
+ * {@link Breadcrumb}s) so that data may be collected in different parts
+ * of an application and then sent together when (e.g.) an exception occurs.
```

Comment:
I suck at English, I'm trying to say that sending breadcrumbs on an exception is technically only an example. Breadcrumbs could be collected to be sent at any time a user desires.